.gitignore

```diff
@@ -25,6 +25,7 @@ cypress/downloads/
 
 # production
 /build/
+/dist/
```

Comment:
In Vite, the build output is `dist/` by default. You’ll see this change reflected in a few places.